src/cogs/utils.py

```diff
@@ -140,6 +140,57 @@ async def verify(self, ctx):
             )
 
     # == START MOD COMMANDS == #
+
+    @commands.command(brief='Create polls using embeds and reactions')
+    @commands.has_any_role('Mods', 616448412057075768)  # Only people with the role of Mods can use this command
+    async def poll(self, ctx, question, *args):
+        """
+        Create embed polls with one question and up to 9 responses.
+        """
+        # Curernt max amount of answers is 9 as there are only 9 digit reactions
+
+        # Define a custom exception that inherits from the base exception
+        class tooLong(Exception):
+            pass
+
+        user = ctx.author  # Get the author for the footer of the embed
+
+        embed = discord.Embed(title=f"**__{question}__**")  # Set the title of the embed as the question provided
+        embed.set_thumbnail(url=user.avatar_url)  # Set the thumbnail as the authors discord profile picture
+        embed.set_author(name=user)  # Set the author of the embed as the person who sent the command
+
+        # Initalize two variables for list comprehension
+        y = 0
+        z = 0
+
+        opts = []  # Use a list to store all the possible options or answers
+        reactions = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣']  # A list of all the reactions that could be added
+
+        # For every possible answer provided appened it to the list for answers
+        for arg in args:
+            opts.append(arg)
+
+        try:
+            # Check how many answers the user provided. If they provided more than 9, raise an exception
+            if len(opts) > 9:
+                raise tooLong
```

Comment:
Why not just check this with an if at the start of the command, and return instead of raising an exception?